test/common/grpc/codec_test.cc

```diff
@@ -106,6 +107,41 @@ TEST(GrpcCodecTest, decodeInvalidFrame) {
   EXPECT_EQ(size, buffer.length());
 }
 
+// A frame of nothing followed by a frame of 0's followed then invalid
```

Comment:
I'm having trouble understanding this test. 
Is a "frame of nothing" a 0 sized frame? It sounds like there are 3 frames in this comment, but it sounds like theres only two by the test name

Reply:
Yeah, "frame of nothing" meaning a frame with zero length and nothing in it. Will reword throughout test to make more clear :).

Reply:
Changed test name to "MultipleInvalidFramesDecoded"

Reply:
Yeah, I think the key for this test is letting code owner and developers know about this null byte frame functionality, and if it is supposed to actually decode null bytes into a frame. I'll reword and be verbose here.

Reply:
I'm gonna add a todo here asking if this test should really pass.